lnprototest/namespace.py

```diff
@@ -18,6 +18,12 @@ def peer_message_namespace() -> MessageNamespace:
                 + pyln.spec.bolt7.csv)
     return ns
 
+def namespace() -> MessageNamespace:
```

Comment:
In a point of view of python code style and also remain under the same stale of `lnprototest` that has a lot of class, do you think that this namespaces module colud be better under a class?

A possible implementation could be with a NamaspacesManager coded as singleto with the `event_namespace` as the propriety of the instance? 

p.s. this is only my idea